src/main/java/org/kiwiproject/dropwizard/util/startup/PortRangeInfo.java

```diff
@@ -0,0 +1,27 @@
+package org.kiwiproject.dropwizard.util.startup;
+
+import static com.google.common.base.Preconditions.checkState;
+import static org.kiwiproject.base.KiwiPreconditions.requireValidPort;
+
+import lombok.Getter;
+
+@Getter
+public class PortRangeInfo {
```

Comment:
Do we want to maybe change this to `AllowablePortRange`?

Reply:
...and docs?

Reply:
sure and sure

Reply:
done